src/control/common/loglevel_test.go

```diff
@@ -0,0 +1,126 @@
+//
+// (C) Copyright 2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package common
+
+import (
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/logging"
+	"github.com/pkg/errors"
+)
+
+func TestCommon_ControlLogLevel_String(t *testing.T) {
+	for name, tc := range map[string]struct {
+		level     ControlLogLevel
+		expResult string
+	}{
+		"error": {
+			level:     ControlLogLevelError,
+			expResult: logging.LogLevelError.String(),
+		},
+		"info": {
+			level:     ControlLogLevelInfo,
+			expResult: logging.LogLevelInfo.String(),
+		},
+		"debug": {
+			level:     ControlLogLevelDebug,
+			expResult: logging.LogLevelDebug.String(),
+		},
+		"unknown": {
+			level:     ControlLogLevel(0xffff),
+			expResult: "UNKNOWN",
+		},
+	} {
+		t.Run(name, func(t *testing.T) {
+			AssertEqual(t, tc.expResult, tc.level.String(), "")
+		})
+	}
+}
+
+func TestCommon_ControlLogLevel_MarshalYAML(t *testing.T) {
+	for name, tc := range map[string]struct {
+		level     ControlLogLevel
+		expResult string
+	}{
+		"error": {
+			level:     ControlLogLevelError,
+			expResult: logging.LogLevelError.String(),
+		},
+		"info": {
+			level:     ControlLogLevelInfo,
+			expResult: logging.LogLevelInfo.String(),
+		},
+		"debug": {
+			level:     ControlLogLevelDebug,
+			expResult: logging.LogLevelDebug.String(),
+		},
+		"unknown": {
+			level:     ControlLogLevel(0xffff),
+			expResult: "UNKNOWN",
```

Comment:
Could be better to define a constant for "UNKOWN" string as for the other log level ?